packages/tests/src.ts/test-wallet.spec.ts

```diff
@@ -539,3 +579,52 @@ describe("Wallet local calls", async function () {
         }
     });
 });
+
+describe("Wallet createAccount", function () {
+
+    let wallet: ethers.Wallet, newAccount: ethers.Wallet, newAccountPublicKey: BytesLike, provider: ethers.providers.BaseProvider;
+
+    before( async () => {
+        const account = {
+            "operator": {
+                "accountId": "0.0.1065",
+                "publicKey": "302a300506032b65700321006286f3cfa771a803f1ff90a3ee5d227002ac209d934f7b47fa41288e71938095",
+                "privateKey": "302e020100300506032b657004220420f24f8a15fc36ec3cee05c99c2e71bda086977885eeeebbec17654c5d3a6c35b5"
+            },
+            "network": {
+                "35.231.208.148:50211": "0.0.3",
+                "35.199.15.177:50211": "0.0.4",
+                "35.225.201.195:50211": "0.0.5",
+                "35.247.109.135:50211": "0.0.6"
+            }
+        };
+
+        provider = ethers.providers.getDefaultProvider('previewnet');
```

Comment:
We should be using `testnet` instead of `previewnet` since previewnet is being cleared. Those tests will start failing in a couple of days